invokeai/backend/stable_diffusion/extensions_manager.py

```diff
@@ -51,6 +64,16 @@ def run_callback(self, callback_type: ExtensionCallbackType, ctx: DenoiseContext
         for cb in callbacks:
             cb.function(ctx)
 
+    def run_override(self, override_type: ExtensionOverrideType, orig_function: Callable[..., Any], *args, **kwargs):
```

Comment:
at least to allow extension manager to run original non-overriden implementation

Reply:
Including it in the function signature implies that the function / override should handle it.

For the case that you're describing, I imagined that it would just look like this:

```python
from ... import unet_forward

class AnExtension(ExtensionBase):
    @override(ExtensionOverrideType.UNET_FORWARD)
    def custom_unet_forward(self, ...):
        # Do some stuff...
        unet_forward(...)
        
```

What do you think?